### PR TITLE
gce: support Ubuntu 20.04LTS image

### PR DESCRIPTION
**Pull request by syuu1228**
Ported Ubuntu AMI code to GCE.

----

Add support Ubuntu 20.04LTS image for GCE.

### CODE REVIEW

**Comment by syuu1228** [comment]
Not directly related with Ubuntu image, it seems like gce/image/build_image.sh, gce/image/files/scylla_install_image are copied from aws/ami, and difference from original is few, maybe we can merge them into single script.
It can take option to specify target cloud, or create symlink from aws/ami to gce/image and then detect target by called filename.

**Comment by syuu1228** [comment]
ping

**Comment by syuu1228** [comment]
@bentsi Please review this

**Comment by bentsi** [comment]
@syuu1228 can you create an image and send me the id so I can run an artifact test on it?

**Comment by syuu1228** [comment]
@bentsi Here's built image:
name: scylla--build-2021-03-22t11-26-26
location: europe-west1

**Comment by bentsi** [comment]
test passed https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/bentsi-staging/job/artifacts-gce-image/21/console

File: gce/image/build_image.sh [hunk 1]
```diff
@@ -105,6 +105,9 @@ check_rpm_exists () {
     done
 }
 
+SOURCE_IMAGE_FAMILY="centos-8"
```
Comment (bentsi) [inline gce/image/build_image.sh:108]: why it is centos-8?

**Comment by syuu1228** [comment]
@bentsi Sorry, I realized gce/image/build_deb_image.sh wasn't added to this PR. Just added and pushed.

**Review by bentsi — APPROVED** [review]

**Pull request merged**